webpack.dev.config.js

```diff
@@ -27,8 +27,12 @@ var cf = {
 			exclude: /node_modules/,
 			loaders: ["raw-loader"]
 		},{
+			test: /\/page1.*\.html$/,
```

Comment:
The issue with ngtemplate-loader is that it puts all of its template loading code into the run  (initialize) block of a angular module called "ng".

Since this code is being lazy loaded, that angular module has already been initialized. Thus the templates cant be added to the init block when they are lazy loaded using ngtemplate-loader.  There might be a way to get the loader to work, but I don't know enough about it. I know the html-loader works though.

The pasted example shows what ngtemplate generates.  The problematic part for lazy loaded content is: ``

When it generates that line during lazy loading, the angular module has already been initialized.